libraries/AP_HAL_ChibiOS/hwdef/CarbonixF405/hwdef.dat

```diff
@@ -120,7 +120,7 @@ define HAL_PERIPH_ENABLE_BARO
 define HAL_PERIPH_ENABLE_RC_OUT
 
 # enable ESC control
-define HAL_SUPPORT_RCOUT_SERIAL 1
```

Comment:
an #ifdef with a #error for the old define will save some devs with out of tree boards some time